doc/dev/index.rst

```diff
@@ -165,6 +165,29 @@ Building from source
 
 See instructions at :doc:`/install/build-ceph`.
 
+Using ccache to speed up local builds
+-------------------------------------
+
+`ccache`_ can speed up rebuilds significantly, since `make clean` is a common
```

Comment:
@theanalyst Looks good; just one nit: "These rebuilds can significantly benefit **from** the use of ccache."